packages/app-project/pages/project.js

```diff
@@ -1,15 +1,68 @@
+import { Grommet } from 'grommet'
+import { Provider } from 'mobx-react'
+import { getSnapshot } from 'mobx-state-tree'
+import PropTypes from 'prop-types'
+import React from 'react'
+
 import Head from '../components/head'
 import Nav from '../components/nav'
-import React from 'react'
-import { Grommet } from 'grommet'
+import ProjectHome from '../components/ProjectHome'
+import getProjectSlugFromURL from '../helpers/getProjectSlugFromURL'
+import initStore from '../stores'
+
+class ProjectPage extends React.Component {
+  static async getInitialProps ({ req }) {
```

Comment:
It might be helpful for a quick comment here to mention that this is part of next.js and not a react lifecycle method (it's just so similarly named to the old `getDefaultProps` that it might be a gotcha for new to next.js folks).

Reply:
👍